.github/workflows/roll_torch_mlir.yml

```
@@ -0,0 +1,60 @@
+name: Roll PyTorch
+
+on:
+  schedule:
+    - cron: '0 13 * * *'
+  workflow_dispatch:
+
+jobs:
+  build_linux:
+    name: roll torch-mlir
+    needs: setup
+    runs-on:
+      - self-hosted
+      - runner-group=${{ needs.setup.outputs.runner-group }}
+      - environment=${{ needs.setup.outputs.runner-env }}
+      - cpu
+      - os-family=Linux
+
+    # Don't run this in everyone's forks.
+    if: github.repository == 'open-xla/iree'
+
+    steps:
+
+      - name: Prepare workspace
+        run: |
+          # Clear the workspace directory so that we don't run into errors about
+          # existing lock files.
+          sudo rm -rf $GITHUB_WORKSPACE/*
+
+      - name: Get iree
+        uses: actions/checkout@v3
+        with:
+          submodules: 'true'
+          token: ${{ secrets.WRITE_ACCESS_TOKEN }}
+          ref: main
+          fetch-depth: 0
+
+      - name: update torch_mlir submodule
```

nit: For consistency
```suggestion
      - name: Update torch_mlir submodule
```